src/services/api.ts

```diff
@@ -44,6 +55,10 @@ export class Api {
             },
             body: data ? JSON.stringify(data) : null,
         })
+
+        if (response.status !== 200 && response.status !== 204) {
```

Comment:
I believe fetch will throw if received error. We can also used `response.ok`,  all http error code >= 200, < 400 are considered valid (FYI)

Reply:
I could print the error result, so I guess it does not throw an error, I also created #393 for this reason. If you agree with #393, this code will be replaced anyway